notebooks/demo3/demo3-create-tables.ipynb

```diff
@@ -0,0 +1,546 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    obj = s3.get_object(
</pre><p> </p><p>Maybe the helper function could be used?</p><pre class="ql-syntax" spellcheck="false">preds_kpi = s3comm.download_df_from_s3(
&nbsp;&nbsp;s3_prefix=config.BASE_INFER_KPI_S3_PREFIX,
&nbsp;&nbsp;s3_key="sustainability-report-2019_predictions_kpi.csv",
&nbsp;&nbsp;filetype=S3FileType.CSV,
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>will replace this! thanks</p>